app/views/pages/hiring_fundraiser.html.haml

```diff
@@ -0,0 +1,120 @@
+      %p.text-p-xlarge.mb-16
+        Exercism is a not-for-profit organisation providing world-class programming education to anyone for free, through a mixture of self-directed learning and mentoring.
+        We have grown to nearly a million members, purely by word-of-mouth, and are recognised as one of the leading education platforms online.
+        Exercism is built by a mixture of our core team and thousands of volunteers.
+      %p.text-p-xlarge.mb-16
+        To take our next steps we need to grow our core team, and to achieve that we need to place ourselves on more solid financial ground.
+        We're looking for someone to lead this process, to work with the CEO to create a fundraising strategy, and then to execute it, raising funds through sponsorship and partnerships.
+        Our target is to generate £500k of revenue in the next 12 months.
+      %p.text-p-xlarge.mb-32
+        Our values of kindness, empathy and personal-growth are core to everything we do - as students, as a community, and as a core team.
+        We're building an exceptional team, full of ambitious, passionate people who excel at what they do, love their work, and who want to honour the commitment of our thousands of volunteers.
+        If that sounds exciting to be part of, and you feel you have the skills to help, we'd love you to apply.
+
+      %h2.text-h2.mb-12 Responsibilities
+      %p.text-p-xlarge.mb-16
+        So far, fundraising has been done on an ad-hoc basis by our CEO.
+        Over the last 3 years, we have raised ~£500k in supprot from organisations.
+        Your role is to take that ad-hoc work, formalise it into a coherent strategy, and then execute on that to deliver £500k of new support in the next 12 months.
+        If the initial work is successful, we can reinvest some of the revenue into growing a sponsorship team around you.
+        You will work closely with the CEO as needed to strategise and explore opportunities.
+      %p.text-p-xlarge.mb-12
+        The key areas of your work will be:
+
+      %ul.text-p-xlarge.mb-40
+        %li Work with the CEO to develop an overarching fundraising strategy.
+        %li Identify and apply for grants.
+        %li Identify, approach, and close deals with potential sponsors.
+        %li Identify partnership opportunities and work with the CEO to close them.
+
+      %h2.text-h2.mb-12 Skills, Background &amp; Qualifications
+      %p.text-p-xlarge.mb-12
+        There are no specific academic qualifications required for this role.
+        The primary criteria for deciding on the right candidate will be:
+      %ul.text-p-xlarge.mb-32
+        %li Experience in fundraising or sales.
+        %li Evidence of an ability to find and close deals.
+        %li A clear ability to think and plan strategically.
+        %li Cultural fit
+
+      %h2.text-h2.mb-12 Other Requirements
+      %p.text-p-xlarge.mb-16
+        We are a fully remote team, who use English as our communication language.
```

Comment:
Typo?
```suggestion
        We are a fully remote team, who uses English as our communication language.
```

Or could be entirely simplified to 
"We are a fully remove team that communicates in English."